core/controllers/topic_editor.py

```diff
@@ -49,6 +50,59 @@ class TopicEditorStoryHandler(base.BaseHandler):
     """
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'topic_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'POST': {
+            'title': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_valid_story_title'
+                    }]
+                }
+            },
+            'description': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            },
+            'filename': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_valid_file_name'
+                    }]
+                }
+            },
+            'thumbnailBgColor': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default': None
```

Comment:
Why is this `None`?

Reply:
I thought attaching image is optional, but when I take a closer look we throw validation error anyway, I removed the none defaults

Reply:
Done.